babble/src/main/java/io/mosaicnetworks/babble/utils/RandomString.java

```diff
@@ -33,6 +33,14 @@
 
 public class RandomString {
 
+    private static final String sUpper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
```

Comment:
Should this just be a hardcoded string? Rather than have 3 constants to build one string. Almost certainly not worth the overhead of changing it, but it bothered me.

Reply:
I agree, it's bothering me now. I'll change it.